src/lib.rs

```diff
@@ -266,25 +326,25 @@ fn open_browser_internal(browser: Browser, url: &str) -> Result<ExitStatus> {
     target_os = "openbsd"
 ))]
 #[inline]
-fn open_browser_internal(browser: Browser, url: &str) -> Result<ExitStatus> {
+fn open_browser_internal(browser: Browser, url: &str, suppress_output: bool) -> Result<ExitStatus> {
```

Comment:
Best to take in `BrowserOptions` instead of `suppress_output` as I mentioned earlier.